frontend/src/app/shared/scss/_colors.scss

```diff
@@ -1,6 +1,8 @@
 /* Variables */
 $main-color: #6a983c;
 $main-dark-color: #46760a;
```

Comment:
schimba-le si pe astea in color-main si color-main-dark , ca sa pastram structura de 

color-ceva